policies/postgres/init.go

```diff
@@ -67,9 +67,10 @@ func migrateDB(db *sqlx.DB) error {
 						valid			 BOOLEAN,
 						agent_group_id	 UUID,
 						agent_policy_id  UUID,
-						sink_id			 UUID,
+						sink_id		     UUID[],
```

Comment:
again let's be consistent and call it `sink_ids` or `sink_id_list` (not sure if there is a precedent in other SQL)